client/components/jetpack/jetpack-product-info/regular-list.tsx

```diff
@@ -1,14 +1,55 @@
-import { FunctionComponent, ReactNode } from 'react';
+import { useRef, useEffect, type FC, type ReactNode } from 'react';
 
 type Props = {
 	items: ReactNode[];
 };
 
-const JetpackProductInfoRegularList: FunctionComponent< Props > = ( { items } ) => {
+const JetpackProductInfoRegularList: FC< Props > = ( { items } ) => {
+	const listRef = useRef< HTMLUListElement | null >( null );
+
+	const highlightListItem = ( target: HTMLElement ) => {
+		target.animate(
+			[ { backgroundColor: 'var(--studio-yellow-5)' }, { backgroundColor: 'initial' } ],
+			{
+				duration: 500,
+				easing: 'ease-out',
+			}
+		);
+	};
+
+	// Highlight list item when its content changes
+	useEffect( () => {
+		if ( ! listRef.current ) {
+			return;
+		}
+
+		const observer = new MutationObserver( ( record ) => {
+			for ( const mutation of record ) {
+				if ( mutation.type !== 'characterData' ) {
```

Comment:
Is this check necessary, given that we're observing only for `characterData` changes anyway?

Reply:
I think you're right, I had all types of mutations being checked at first before I cut it down to only characterData, this check is probably not necessary